packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -100,15 +100,10 @@ export class AadAppForTeamsImpl {
     await TokenProvider.init({ m365: ctx.m365TokenProvider });
 
     // Move objectId etc. from input to output.
-    const skip = Utils.skipAADProvision(
-      ctx,
-      isLocalDebug ? (isConfigUnifyEnabled() ? false : true) : false
```

Comment:
Still need the check for isLocalDebug?

Reply:
No, I think `Utils.skipAADProvision` can be updated later because local and remote are now using the same configs.